lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-beta.32",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "5.0.0"
```

Comment:
yes. I was thinking we'd use lerna in `integrated` mode, which increments all versions together (rather than keeping a compatibility matrix in our head)